### ISSUE TITLE
distance calculation with x/z selector

### ISSUE THREAD

**Issue by BuckarooBanzay** [minetest-mapserver/mapserver#158]

**Comment by MilesBDyson** [comment]
yes a ruler would be nice. able to place more then one would be better. this wold be so handy for planning out future large builds as well as aiding in building large builds and measuring how close your neighbors are !!
